.gitignore

```diff
@@ -0,0 +1,11 @@
+node_modules/
+dist/
+venv/
+tmp/
+*.log
+.idea
+.vscode/
+.floo
+.flooignore
+package-lock.json
+.DS_Store
```

Comment:
Check this comment: https://github.com/OpenPaaS-Suite/esn-frontend-calendar/pull/5/files#r450718701

Reply:
Thanks, I updated it